hack/pre-release/pre-release.sh

```
@@ -0,0 +1,279 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$0")" >/dev/null || exit
+    pwd
+)"
+
+PROJECT_DIR="$(
```

Shellcheck (0.11.0)

[warning] PROJECT_DIR appears unused. Verify use (or export if used externally).

(SC2034)